python/src/trezorlib/cardano.py

```diff
@@ -262,7 +280,7 @@ def _parse_address_parameters(
     )
 
     return create_address_parameters(
-        int(address_parameters["addressType"]),
+        messages.CardanoAddressType(int(address_parameters["addressType"])),
```

Comment:
please try removing the `int()` inside, see what happens

Reply:
Works fine even without it, deleted in b70acaf